config/wdio/wdio.conf.js

```diff
@@ -11,7 +12,11 @@ const visualRegressionConfig = require('./visualRegressionConf');
 const TerraWDIOSpecReporter = require('../../reporters/wdio/TerraWDIOSpecReporter');
 
 /* Use to pass your host's IP when running wdio tests from a VM or behind a proxy. */
-const ip = process.env.WDIO_EXTERNAL_HOST || localIP.address();
+let ip = process.env.WDIO_EXTERNAL_HOST || localIP.address();
+const networkInterfaces = os.networkInterfaces();
+if (!process.env.WDIO_EXTERNAL_HOST && networkInterfaces.utun2 && networkInterfaces.utun2[0]) {
+  ip = networkInterfaces.utun2[0].address;
+}
```

Comment:
can this be simplified to 👇 ...???
```suggestion
const networkInterfaces = os.networkInterfaces();
const ip = process.env.WDIO_EXTERNAL_HOST || networkInterfaces?.utun2 && networkInterfaces.utun2[0]?.address || localIP.address();
```

Reply:
Have you tried it? Because on my machine the chaining operator throws up.